app/src/main/kotlin/com/simplemobiletools/musicplayer/helpers/MultiPlayer.kt

```diff
@@ -0,0 +1,281 @@
+package com.simplemobiletools.musicplayer.helpers
+
+import android.app.Application
+import android.content.IntentFilter
+import android.media.AudioAttributes
+import android.media.AudioManager
+import android.media.MediaPlayer
+import android.net.Uri
+import android.os.PowerManager
+import androidx.core.content.getSystemService
+import androidx.media.AudioAttributesCompat
+import androidx.media.AudioFocusRequestCompat
+import androidx.media.AudioManagerCompat
+import com.simplemobiletools.musicplayer.extensions.config
+import com.simplemobiletools.musicplayer.receivers.HeadsetPlugReceiver
+
+class MultiPlayer(private val app: Application, private val callbacks: PlaybackCallbacks) : MediaPlayer.OnErrorListener, MediaPlayer.OnCompletionListener {
+    private var mCurrentMediaPlayer = MediaPlayer()
+    private var mNextMediaPlayer: MediaPlayer? = null
+
+    var isInitialized: Boolean = false
+        private set
```

Comment:
this look weird, shouldnt it be a casual variable?

Reply:
sure